CVInHtml/cv.html

```diff
@@ -5,115 +5,134 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>CV exercise</title>
+    <link rel="stylesheet" href="bootstrap/css/bootstrap.min.css">
```

Comment:
Let's add also physical bootstrap file

Reply:
The file bootstrap.min.css has been added upon suggestion

Reply:
File added and changed file source from bootstrap/css/bootstrap.min.css to bootstrap/bootstrap.min.css